src/library/templates/library/partial/library_card.html

```diff
@@ -1,6 +1,6 @@
 {% load static library %}
 <div class="card card-library">
-    <div class="card-header">
+    <div class="card-header has-favorite" >
```

Comment:
Class attribute should presumably only be included if show_starred is true.

Reply:
Both library card mockups have the has-favorite class whether the current state is checked or not.

Reply:
`.has-favorite` is needed to give the title a margin on the right side to keep the text from flowing under the star button.  Needs to be there only if the star button is present.  Not using the check here adds the unneeded margin on the dashboard cards.

Reply:
I missed this the first time through - now class has-favorite is not used when the starred value is not present (e.g., dashboard).